apps/trading/e2e/README.md

```diff
@@ -0,0 +1,118 @@
+# Trading End-To-End Tests
```

Comment:
This comment relates to the folder hierachy and naming. We have trading-e2e and trading/e2e, we could perhaps distinguish a bit more between the two as it is a little confusing.